subscription/src/main/java/org/killbill/billing/subscription/api/SubscriptionBaseApiService.java

```diff
@@ -107,4 +108,9 @@ public List<SubscriptionBaseEvent> getEventsOnCancelPlan(final DefaultSubscripti
                                                              final InternalTenantContext internalTenantContext) throws CatalogApiException;
 
     boolean undoChangePlan(DefaultSubscriptionBase defaultSubscriptionBase, CallContext context) throws SubscriptionBaseApiException;
+
+    int getAccountBCD(InternalTenantContext context) throws SubscriptionBaseApiException;
+
+    // If we expose more than one config, we should not instead return the SubscriptionConfig object
```

Comment:
Is this a question? 

BTW I don't think you need the `@Param("dummy")` here.

Reply:
Fixed the comment and removed the unused dummy param